src/ergw_aaa_diameter_srv.erl

```diff
@@ -179,7 +179,8 @@ call(App, Request, #{function := Function} = Config, CallOpts) ->
 	    case Error of
 		encode ->
 		    ?LOG(critical, "failed to encode DIAMETER requests: ~0p", [Request]);
-		_ ->
+		Other ->
+		    ?LOG(notice, "non-critical diameter API: ~0p", [Other]),
 		    ok
```

Comment:
The `lagger` will return atom `ok` - as in line 181 where is used log - the `ok` can removed`.

Reply:
Ok